src/meta/src/hummock/manager/mod.rs

```diff
@@ -495,11 +495,12 @@ where
                 return Err(Error::InvalidContext(context_id));
             }
         }
-        trx.check_equal(
-            META_CF_NAME.to_owned(),
-            META_LEADER_KEY.as_bytes().to_vec(),
-            info.encode_to_vec(),
-        );
+        // Temporarily disabling because of https://github.com/risingwavelabs/risingwave/issues/6534
+        // trx.check_equal(
```

Comment:
The new leader info should be stored in `META_LEADER_KEY` before it become leader and start the hummock manager. We might need to figure out why and fix it rather than ignore it.

Reply:
Resolving it in a follow-up PR is acceptable for me. Please add a `FIXME` for it.